- Removed CDN link.

elliottmejia opened the pull request:
- Added Tailwind CSS as a dependency.
- Integrated Tailwind CSS plugin in vite.config.ts for styling support. Used PluginOption[] assertion to tell our typescript overlord the obvious.
- Created a global.css file for tailwind to target.
- Refactored contents of <style/> in  index.html -> global.css file for cleanliness.

elliottmejia committed:
- Removed CDN link.
- Added Tailwind CSS as a dependency.
- Integrated Tailwind CSS plugin in vite.config.ts for styling support. Used PluginOption[] assertion to tell our typescript overlord the obvious.
- Created a global.css file for tailwind to target.
- Refactored contents of <style/> in  index.html -> global.css file for cleanliness.

WilsonNet commented:
Thanks @elliottmejia, it is not passing on the linter right now. Feel free to fix it if you are there,  otherwise I will send a commit fixing to this pr and merge with squash

WilsonNet committed:
fix: resolve linter and validation issues

- Add type annotation to PluginOption import in vite.config.ts
- Remove duplicate rel attribute from stylesheet link
- Remove invalid script tag and noscript with invalid alert element
- All CI checks now pass (lint, format, build)
